{"uuid": "f2efc566-8c17-4158-af14-d1872f8a9b64", "children": ["87c1e242-3d19-4223-b7ff-db90d4d65403", "9e6ef2c1-df82-430f-b61a-2e8b42367a12", "4e4b0e59-88d3-4782-9e6c-d9ae50c2e049", "5c5332ba-aa7e-491b-8e7a-2ce649eb6175", "a24742ca-88a8-424d-acec-5d89e24c9073", "23444a9d-17da-4fd4-9e41-9c8d77d3197b"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988374456, "stop": 1699988374456}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988374465, "stop": 1699988374465}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988374471, "stop": 1699988374471}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988374477, "stop": 1699988374477}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988374483, "stop": 1699988374483}, {"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._api_types.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/kamaz/PycharmProjects/FinalProject/conftest.py\", line 26, in browser\n    with sync_playwright() as playwright:\n  File \"/home/kamaz/PycharmProjects/FinalProject/venv/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py\", line 47, in __enter__\n    raise Error(\n"}, "start": 1699988374488, "stop": 1699988374488}], "start": 1699988374488, "stop": 1699988374493}